driver/driver.go

```diff
@@ -50,8 +50,41 @@ type ResolvedCred struct {
 
 // OperationResult is the output of the Driver running an Operation.
 type OperationResult struct {
-	// Outputs is a map from the container path of an output file to its contents (i.e. /cnab/app/outputs/...).
+	// Outputs maps from the name of the output to its content.
 	Outputs map[string]string
+
+	// Logs is the combined logs from the bundle execution.
+	Logs bytes.Buffer
+
+	// Error is any errors from executing the operation.
+	Error error
+}
+
+// SetDefaultOutputValues for an output when it does not exist and it has a
+// non-empty default value.
+func (r *OperationResult) SetDefaultOutputValues(op Operation) error {
```

Comment:
I may have missed a unit test for this function - is there one already?

Reply:
I thought I had a couple but I can't find them anymore! 😱  I had them at the Run level to make sure they were being called (in TestAction_RunAction). I'll hunt them down from my git ref log and resurrect them.

Reply:
Found and pushed!